includes/definitions/proxmox.yaml

```diff
@@ -21,5 +21,7 @@ discovery_modules:
     libvirt-vminfo: 0
 processor_stacked: 1
 discovery:
-    - sysDescr_regex:
-        - '/PVE/'
+    - sysObjectId:
```

Comment:
This is wrong, it means match either sysObjectID or sysDescr.   You need to have them in the same array (-).